src/include/simeng/pipeline/ExecuteUnit.hh

```diff
@@ -65,6 +65,8 @@ class ExecuteUnit {
   /** Retrieve the number of active execution cycles. */
   uint64_t getCycles() const;
 
+  bool isEmpty() const;
```

Comment:
Comment needed here

Reply:
I have taken the solution from #298 as this check is correct